monday.com/1.0.0/README.md

```diff
@@ -0,0 +1,81 @@
+# Monday.com Integration
+
+Create Monday.com Items from Keptn
+
+## Step 1: Gather Monday.com Details
+1. Go to Monday.com and copy your Personal API token. "User Circle Icon" > Admin > API
+  
+![monday.com api](/assets/1.png)
```

Comment:
As described in #104 Artifacthub currently doesn't support relative paths. The reason is that when processing packages AH only reads the metadata files. They don't store or serve any other files. Therefore all the relative paths in the PR should be changed to absolute paths. Will also add this fact to the README to avoid future problems.